src/exometer_report.erl

```diff
@@ -1149,20 +1160,21 @@ cancel_subscr_timers(Reporter) ->
 					       _ = '_'},
 				    _ = '_'}, [], ['$_']}])).
 
-restart_subscr_timer(Key, Interval) when is_integer(Interval) ->
-    TRef = erlang:send_after(Interval, self(),
-			     {report, Key, Interval}),
+restart_subscr_timer(Key, Interval, T0) when is_integer(Interval) ->
+    TRef = erlang:send_after(adjust_interval(Interval, T0), self(),
+			     {report, Key, Interval, os:timestamp()}),
```

Comment:
Don't you think `os:timestamp()` should be substituted with `T0` here? By calling `os:timestamp()` we record the time when the `send_after/3` function is called and this may be much later than the event of timer firing. And we want to count the next interval from the latter.